draft-ietf-scitt-architecture.md

```diff
@@ -326,6 +326,16 @@ The SCITT architecture consists of a very loose federation of Transparency Servi
 In order to accommodate as many Transparency Service implementations as possible, this document only specifies the format of Signed Statements (which must be used by all Issuers) and a very thin wrapper format for Receipts, which specifies the Transparency Service identity and the agility parameters for the Signed Inclusion Proofs.
 Most of the details of the Receipt's contents are specified in the COSE Signed Merkle Tree Proof document {{-COMETRE}}.
 
+{{fig-concept-relationship}} illustrates two Transparency Services and two Receipts as a single Signed Statement MAY be registered on more than one Transparency Service.
+Each Transparency Service produces a Receipt.
+A Transparent Statement MAY contain multiple Receipts, for each Transparency Service it was successfully Registered.
+
+The arrows indicate the flow of information.
+For example, Credentials are used to produce Signed Statements and the Transparency Service verifies the Signed Statements.
+The Transparency Service produces a Receipt.
+A Receipt combined with a Signed Statement creates a Transparent Statement.
+To verify a Transparent Statement, credentials are required to authenticate the protected content of the Transparent Statement.
```

Comment:
Wouldn't that mean that you are actually only checking the signature of the Signed Statement, as the "unprotected content" is not checked?

Reply:
Resolved with committed suggestion